lib/errors.ts

```diff
@@ -14,18 +14,76 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import errors = require('resin-cli-errors');
-import patterns = require('./utils/patterns');
+import _ = require('lodash');
+import os = require('os');
```

Comment:
Why not
```typescript
import * as os from 'os';
```
?

Reply:
No real reason, it's just that that's what the existing imports did. I've now updated all of them.